src/Components/footerNav.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter as Router, NavLink } from 'react-router-dom';
+
+function Footer() {
+  return (
+    <Router>
```

Comment:
It may not be necessary to wrap this with `<Router />`, since you have context of the router via `<Router />` being used in `<App />` already.